src/routes/publicRoutes/index.js

```diff
@@ -72,8 +75,8 @@ export default function PublicRoutes() {
         <>
             {isAuthenticated && !isProcessingPage && !isStripePage && !isSuccessPage && !isErrorPage && <Navbar />}
             <Routes>
-                <Route exact path={routes.LOGIN} element={<Login />}></Route>
-                <Route exact path={routes.SIGNUP} element={<SignUp />}></Route>
+                {!isAuthenticated ? <Route exact path={routes.LOGIN} element={<Login />}></Route> : ""}
```

Comment:
you can remove duplicacy using loop or anothor suitable method in order to optimise this.